### DISCUSSION TITLE
Adding Fn+Insert (Sleep Function)

### DISCUSSION THREAD

**Discussion by quynkk5** [SkyrilHD/Dell-Exx50-Hackintosh#25] [Ideas]
I fixed Fn+Insert for sleep function by adding this SSDT and 4 ACPI rename
Tested in Dell Latitude E5550 and worked properly
And I also support Clover Bootloader Revision 5138 for Dell Latitude E5550, you can find it in my list: https://github.com/quynkk5/Hackintosh-List
[SSDT-BTNV.dsl.zip](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/files/7068900/SSDT-BTNV.dsl.zip)

**Comment by SkyrilHD** [comment]
I don't get what you mean. FN + Insert works perfectly fine here. You don't need any patches. Could be because you set up your Clover configuration wrong
